syscall.go

```diff
@@ -115,10 +117,18 @@ func unmountShareDir() error {
 	return os.RemoveAll(containerMountDest)
 }
 
-func waitForBlockDevice(deviceName string) error {
+func waitForBlockDevice(deviceName string, isSCSIAddr bool) error {
```

Comment:
You should rename this function into `waitForDevice()` and pass a more global structure. Once again, this looks very specific to SCSI here.

Reply:
Any comment on this ?

Reply:
I would prefer keeping this as  is as we are waiting for a Block device here, the difference being if the device is passed with virtio-scsi or virtio-block, with the flag indicating this.

Reply:
Yes I agree this is still a block device, forget this comment.